src/carousel.tsx

```diff
@@ -426,16 +422,15 @@ const BaseCarousel: React.FC<BaseCarouselProps> = ({
         gap,
         centered,
         isDesktopOrBigger,
-        sideMargin,
         items.length,
     ]);
 
     const goToPage = React.useCallback(
-        (pageIndex: number) => {
+        (pageIndex: number, animate = true) => {
             const carouselEl = carouselRef.current;
             if (carouselEl) {
                 const scroll = pagesScrollPositions[pageIndex];
-                carouselEl.scrollTo({left: scroll, behavior: 'smooth'});
+                carouselEl.scrollTo({left: scroll, behavior: animate ? 'smooth' : 'auto'});
```

Comment:
unrelated change, but caused the screenshot diff (not sure why)

Reply:
and who is using the 'true' value?

Reply:
is the default behavior. For example, when clicking on next/prev buttons or in the carousel bullets